### PR TITLE
fix: change gc jobs RBAC roles

### PR DESCRIPTION
**Pull request by rast1025**
It should fix gc jobs permissions error

### CODE REVIEW

File: charts/jxboot-helmfile-resources/templates/jx-gcjobs-role.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```
Comment (ankitm123) [inline charts/jxboot-helmfile-resources/templates/jx-gcjobs-role.yaml:2]: Use role, and create in `jx-git-operator` ns

**Review by ankitm123 — APPROVED** [review]

**Comment by yelhouti** [comment]
this broken:
https://github.com/jx3-gitops-repositories/jx3-kubernetes/blob/62a891b6ce51de4d6713171683d99db981011b56/versionStream/src/Makefile.mk#L127

The command above moves all files to the namespaces where they where generated (based on the Release.namespace)...
Jobs should stay in the jx namespace and not be configurable

**Pull request merged**